### PR TITLE
CI: replace publish image step with a much simpler one that supports LATEST

### PR DESCRIPTION
**Pull request by kminehart**

### CODE REVIEW

**Review by ryantxu — APPROVED** [review]
approve the skip... but not sure the rest :)

File: pkg/services/queryhistory/queryhistory_search_test.go [hunk 1]
```diff
@@ -264,6 +264,7 @@ func TestIntegrationGetQueriesFromQueryHistory(t *testing.T) {
 
 	testScenarioWithMixedQueriesInQueryHistory(t, "When users tries to get queries with mixed data source it should return correct queries",
 		func(t *testing.T, sc scenarioContext) {
+			t.Skip() // This test fails a lot at the moment
```
Comment (ryantxu) [inline pkg/services/queryhistory/queryhistory_search_test.go:267]: @kminehart is this failing in all the backports?  should we make a new PR with only this change?

Comment (ryantxu) [reply pkg/services/queryhistory/queryhistory_search_test.go:267]: https://github.com/grafana/grafana/pull/90553

**Review by zserge — APPROVED** [review]

**Review by baldm0mma — APPROVED** [review]

File: pkg/services/queryhistory/queryhistory_search_test.go (hunk 1 above)
Comment (kminehart) [reply pkg/services/queryhistory/queryhistory_search_test.go:267]: ah ok I will remove that change from this PR. nice!

**Pull request merged**